crates/test-helpers/macros/build.rs

```diff
@@ -168,14 +162,8 @@ fn main() {
             ));
 
             let dep_file = file.with_extension("d");
-            let deps = fs::read_to_string(&dep_file).expect("failed to read dep file");
-            for dep in deps
-                .splitn(2, ":")
-                .skip(1)
-                .next()
-                .unwrap()
-                .split_whitespace()
-            {
+            let deps = fs::read_to_string(dep_file).expect("failed to read dep file");
+            for dep in deps.split_once(':').unwrap().1.split_whitespace() {
```

Comment:
I don't think this transformation is correct - we want the split whitespace after the second `:`